src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/CreateOrganizationOverlay.tsx

```diff
@@ -254,6 +279,11 @@ export const CreateOrganizationOverlay: FC = () => {
       } else {
         setNetworkErrorMsg(OrgOverlayNetworkError.DefaultError)
       }
+
+      event(CreateOrgOverlayEvent.OrgCreationFail, multiOrgTag, {
+        currentOrgId,
+      })
+
```

Comment:
same comment re 'current' vs. 'old'.

Reply:
since this is a case where no new org has been created, doesn't it make more sense to name it `current` ? Calling it old implies there is a new org created. Personally, I think currentOrgID or just orgID is a more accurate and descriptive name than `oldOrgID`. What do you think?